src/NuGet.Core/NuGet.ProjectModel/PackageSpecExtensions.cs

```diff
@@ -43,5 +44,31 @@ public static ProjectRestoreMetadataFrameworkInfo GetRestoreMetadataFramework(th
 
             return frameworkInfo ?? new ProjectRestoreMetadataFrameworkInfo();
         }
+
+        /// <summary>
+        /// Merge the CentralVersion information to the package reference information.
+        /// </summary>
+        /// <param name="project">The package spec to update.</param>
+        /// <returns>The updated package spec.</returns>
+        public static PackageSpec ApplyCentralVersionInformation(this PackageSpec project)
```

Comment:
I see this method returns the same instance of `PackageSpec` that is passed as parameter. Why can't we have this method to return `void`  because the parameter is passed by reference ? Am I missing anything here ?

Reply:
I think returning void is a good idea.

Reply:
I think that the usability was better if returned the object, but I changed to void based on all three comments.